Rakefile.rb

```diff
@@ -12,20 +13,26 @@
 
   desc 'Pull the integrations repo'
   task :'pull-integrations' do
-    sh "cd /#{ENV['INTEGRATIONS_REPO']} &&
-        git fetch --all &&
-        git checkout dd-check-#{ENV['INTEGRATION']}-#{ENV['VERSION']} &&
-        git reset --hard"
+    integration_branch = ENV['VERSION'] || 'master'
+
+    sh "git -C /#{ENV['INTEGRATIONS_REPO']} fetch --all"
```

Comment:
I actually found another way to do it in Ruby: http://stackoverflow.com/a/10148325.
It avoids the repetition of all the `-C` without having to run a `cd`.
Completely a nitpick though.